src/main/java/com/jayway/maven/plugins/android/phase04processclasses/ProguardMojo.java

```diff
@@ -429,7 +442,7 @@ private void executeProguard() throws MojoExecutionException
             IOUtils.write( commandStringBuilder, tempConfigFileOutputStream );
 
             executor.setCaptureStdOut( true );
-            commands.add( "@\"" + tempConfigFile.getAbsolutePath() + "\"" );
+            commands.add( "@" + tempConfigFile.getAbsolutePath() + "" );
```

Comment:
Why did you remove the escaped " .. these are needed on Windows as far as I know.